examples/test.php

```diff
@@ -43,7 +43,7 @@ protected function init()
 
             // look if name is unique
             $c = (clone $m)->unload()->tryLoadBy('name', $m['name']);
-            if ($c->loaded() && $c->id != $m->id) {
+            if ($c->loaded() && $c->id !== $m->id) {
```

Comment:
Actually ID property was dropped, should be ->getId() or ->get('id') or ->get($c->id_field)